packages/vue3-jest/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "vue3-jest",
+  "version": "26.0.0-alpha.10",
```

Comment:
How do you think we should handle this? Should this match a specific Jest version (maybe latest 26.x minor version?)